examples/table/mui/tableFilter/src/pages/posts/list.tsx

```diff
@@ -0,0 +1,274 @@
+                    return <TagField value={row.status} color={color} />;
+                },
+                minWidth: 120,
+                flex: 0.3,
+            },
+            {
+                field: "createdAt",
+                headerName: "Created At",
+                renderCell: function render({ row }) {
+                    return <DateField value={row.createdAt} format="LLL" />;
+                },
+                minWidth: 150,
+                flex: 0.5,
+            },
+            {
+                field: "category.id",
+                headerName: "Category",
+                type: "number",
+                headerAlign: "left",
+                align: "left",
+                minWidth: 250,
+                flex: 0.5,
+                valueGetter: ({ row }) => {
+                    const { data } = useOne<ICategory>({
+                        resource: "categories",
+                        id: row.category.id,
+                    });
+                    return data?.data.title;
+                },
+            },
+        ],
+        [],
+    );
+
+    const { dataGridProps, filters, search } = useDataGrid<
+        IPost,
+        HttpError,
+        IPostFilterVariables
+    >({
+        columns,
+        initialPageSize: 10,
+        onSearch: (params) => {
+            const filters: CrudFilters = [];
+            const { q, category, status, createdAt } = params;
+
+            filters.push(
+                {
+                    field: "q",
+                    operator: "eq",
+                    value: q !== "" ? q : undefined,
+                },
+                {
+                    field: "category.id",
+                    operator: "eq",
+                    value: category !== "" ? category : undefined,
+                },
+                {
+                    field: "status",
+                    operator: "eq",
+                    value: status !== "" ? status : undefined,
+                },
+                // {
+                //     field: "createdAt",
+                //     operator: "gte",
+                //     value: createdAt ? createdAt[0].toISOString() : undefined,
+                // },
+                // {
+                //     field: "createdAt",
+                //     operator: "lte",
+                //     value: createdAt ? createdAt[0].toISOString() : undefined,
+                // },
+            );
+
+            return filters;
+        },
+    });
+
+    const { control, register, handleSubmit } = useForm<
+        IPost,
+        HttpError,
+        IPostFilterVariables
+    >({
+        defaultValues: {
+            q: getDefaultFilter("q", filters, "eq"),
+            status: getDefaultFilter("status", filters, "eq"),
+            category: getDefaultFilter("category.id", filters, "eq"),
+            createdAt: [
+                getDefaultFilter("createdAt", filters, "gte"),
+                getDefaultFilter("createdAt", filters, "lte"),
+            ],
```

Comment:
```suggestion
            createdAt: [
                getDefaultFilter("createdAt", filters, "gte"),
                getDefaultFilter("createdAt", filters, "lte"),
            ],
```
Do we have a date filter?